custom_components/symx-ha-integration/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+	"domain": "symx_ha_integration",
```

Comment:
please update naming to match new repository name. This has also to be done for the custom_components folder.